src/views/home/sections/Sponsors/Sponsors.tsx

```diff
@@ -5,37 +5,72 @@ import Row from "react-bootstrap/Row";
 
 import sponsors from "assets/data/sponsors.json";
 import logo_stickerMule from "assets/sponsors/sticker-mule-logo-light-bg-stacked.svg";
+import logo_cockroach from "assets/sponsors/cockroach_labs.svg";
+import logo_costar from "assets/sponsors/costar_group.svg";
+import logo_googleCloud from "assets/sponsors/google_cloud.svg";
+import logo_mobilityWare from "assets/sponsors/mobilityware.svg";
+import logo_pimco from "assets/sponsors/pimco.svg";
+import logo_twilio from "assets/sponsors/twilio.svg";
+import logo_wayup from "assets/sponsors/wayup.svg";
```

Comment:
A bit of a nitpick: can we remove `"src"` in `/src/assets/data/sponsors.json` since it's now redundant from these imports? Other than that it looks good to me.